### ISSUE TITLE
Strange behavior in a multi monitor setup

### ISSUE THREAD

**Issue by jgarciao** [flathub/org.sugarlabs.AbacusActivity#1]
Hi Martín, congratulations for having this app finally on Flathub!

I just tried it in my computer and it seems to work fine. The only think that I find a little strange is that I have two monitors with different sizes and the app opens maximized using both monitors, resulting a little difficult to use (the abacus is in the middle of both monitors). I couldn't find a way to use just in one of the monitors.

I don't know if it's intended to be like this or just a bug

**Comment by tchx84** [comment]
Hola Jorge, it is definitely not intended to use both monitors... I will try to reproduce it and see what is going on. Thanks!

**Comment by tchx84** [comment]
I tried this out, and I see what you mean now.

Half (or less) of the application shows in one monitor and the other half in the other monitor. Not only makes it difficult to use, but it is a very bad user experience.

It took me some time to understand what was going on, but I should have fixed it now in the PR mentioned above. The most appropriate trade-off I found was to make the applications run in the primary monitor.

But just to future reference, I want to leave here a few notes for the next time we revise this.

* Sugar applications were not designed for working on multiple monitors. They rely on old GDK APIs, like `Gdk.Screen.width()`, which returns the whole sum of the width available by all the joint-monitors.

*  The issue seems present even in other full-screen applications like `com.tux4kids.tuxtype` and `net.supertuxkart.SuperTuxKart`. 

* So, one could argue that there are valid scenarios for this behavior but, IHMO, it is better to ensure that Sugar applications will always be use-able (even if it comes at the cost of preventing someone showing it full-screen in 3 monitor array).
